### PR TITLE
Skip pymapd and pyspark for py38

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jreback** [comment]
what is the purpose of actually building new packages here?

**Comment by xmnlab** [comment]
the installation for python3.8 has unsolved dependencies:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
```

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,22 +6,23 @@ package:
...
   host:
     - pip
-    - python >=3
+    - python
     - setuptools
 
   run:
     - clickhouse-driver >=0.1.3
-    - clickhouse-cityhash
+    - clickhouse-cityhash  # [not win]
```
Comment (xmnlab) [inline recipe/meta.yaml:25]: the feedstock windows tests worked. but not sure how is the impact for the entire backend without this package.
ibis ci doesn't test clickhouse backend

**Comment by xmnlab** [comment]
this PR is ready for review. thanks!

**Review by jreback — CHANGES_REQUESTED** [review]
i am note sure why you did this PR

**Comment by xmnlab** [comment]
1. noarch-python building suppose the same dependencies for each python version and architecture, and for Ibis these conditions are not true.
2. as the dependencies for python 3.8 is not corrected set, when install ibis for python 3.8 the building cannot solve the dependencies (see https://github.com/conda-forge/ibis-framework-feedstock/pull/41#issuecomment-613174035)

**Comment by jreback** [comment]
let’s fix the actual CI first

this is completely orthogonal

**Comment by xmnlab** [comment]
I think it is related to one of the currently problems we have on CI as condabuild job is not working and it is now using python 3.8 https://dev.azure.com/ibis-project/ibis/_build/results?buildId=2276&view=logs&j=179055bf-797c-5905-134a-4a7381d87002&t=e379731c-be9b-5a2e-f039-d0e3931908e6&l=138

**Comment by xmnlab** [comment]
@jreback if there is no problem with this new change, could we have this merged? so I can check if it will fix the CI condabuild job

**Comment by jreback** [comment]
i prefer to fix the actual co first
you changed quite a bit here so this needs a look. as well

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
ok, merging to fix the main CI. this dependency is pretty odd to actually have the feedstock connected to the CI itself in the project.

**Pull request merged**